### PR TITLE
Remove python2 from base_tools.sh

### PR DESCRIPTION
**Pull request by andybui01**
`python-is-python2` provides `python` in apt and installs python2 dependencies, which we no longer need.

See: https://github.com/seL4/docs/pull/188

### CODE REVIEW

File: scripts/base_tools.sh
Comment (axel-h) [inline scripts/base_tools.sh:80]: The failure
```
+ repo init -u https://github.com/seL4/sel4test-manifest.git --depth=1
/usr/bin/env: 'python': No such file or directory
```
should be gone when we install the `python-dev-is-python3` package then also. See
https://packages.debian.org/bullseye/python-dev-is-python3

Comment (andybui01) [reply scripts/base_tools.sh:80]: fixed, thanks

Comment (axel-h) [reply scripts/base_tools.sh:80]: I wonder, maybe the packege `python-is-python3` is enough, this is what I usually use.

Comment (andybui01) [reply scripts/base_tools.sh:80]: you might be right, I thought it was a python-dev specific error but maybe not. I think the dockerifle should reflect the installation process of the docs, let me try python-is-python3.

**Review by lsf37 — APPROVED** [review]
Thanks for that.

The docker install should definitely be in sync with the build instructions on the doc site. In this case I would say that the docs should add `python-is-python3` if they don't do that already.

There is no good way to test if we're really removed all python2 dependencies across all repos, so I'm just going to merge this and we will see if there is any fallout after the first seL4 PR is merged (which triggers tests in all other repos).

**Pull request merged**
